examples/src/Components/RecordMovieComponent.tsx

```diff
@@ -1,13 +1,18 @@
+import { set } from "lodash";
```

Comment:
linter says this is unused - accidental auto-import?

Reply:
sigh